src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -264,8 +266,8 @@
                             [[:div {:style {:float "right"}}
                               [create/Button (select-keys props [:nav-context :billing-projects :disabled-reason])]]
                              [:div {:style {:clear "left" :float "left" :marginTop "0.5rem"}}
-                              (style/create-link {:text (if filters-expanded? "Collapse" "Show additional filters")
-                                                  :onClick #(swap! state update :filters-expanded? not)})]
+                              (links/create-internal {:onClick #(swap! state update :filters-expanded? not)}
+                                                     (if filters-expanded? "Collapse" "Show additional filters"))]
```

Comment:
Can we take a sec and change this text to "Expand filters" & "Collapse filters"?